packages/core/test/collectAsyncCalls.ts

```diff
@@ -2,29 +2,32 @@ import { getCurrentJasmineSpec } from './getCurrentJasmineSpec'
 
 export function collectAsyncCalls<F extends jasmine.Func>(
   spy: jasmine.Spy<F>,
-  expectedCallsCount: number,
-  callback: (calls: jasmine.Calls<F>) => void
-) {
-  const currentSpec = getCurrentJasmineSpec()
-  if (!currentSpec) {
-    throw new Error('collectAsyncCalls should be called within jasmine code')
-  }
+  expectedCallsCount: number
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ make `expectedCallsCount` default to one, as it seems to be used often